BedrockCore.cpp

```diff
@@ -107,11 +107,14 @@ void BedrockCore::prePeekCommand(unique_ptr<BedrockCommand>& command) {
             SALERT("Command " << command->request.methodLine << " timed out after " << e.time()/1000 << "ms.");
         }
         STHROW("555 Timeout prePeeking command");
+        command->complete = true;
     } catch (const SException& e) {
         _handleCommandException(command, e);
+        command->complete = true;
     } catch (...) {
         SALERT("Unhandled exception typename: " << SGetCurrentExceptionName() << ", command: " << request.methodLine);
         command->response.methodLine = "500 Unhandled Exception";
+        command->complete = true;
```

Comment:
I like the way peek handles these all in one place better, but I don't see an obvious way to do that here. NAB.